scripts/GOOGLE_TTS_TROUBLESHOOTING.md

```
@@ -0,0 +1,93 @@
+
+4. **특정 상황만 재생성**
+   ```bash
+   # 실패한 상황만 다시 생성
+   node scripts/generate-audio-from-callscript.js --api google --situation fire-far
+   ```
+
+## 예상되는 동작
+
+재시도 로직이 포함된 스크립트 실행 시:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)